### ISSUE TITLE
[Smoke Test] Progress bar does not work functionally

### ISSUE THREAD

**Issue by Conjuror** [mozilla-tw/Zerda#334]
Build 439

DuT: Advan

Attempt to load Amazon.com, no progress bar appears on the screen

**Comment by dakaizou** [comment]
improved in build 623, but no fading out
DuT: Samsung J7 Prime, Advan

**Comment by Conjuror** [comment]
As @dakaizou mentioned, it may related to #277 and worth to take to a look.

**Comment by mTwTm** [comment]
Some random log suggesting we probably shouldn't trust onPageStarted and onPageFinished for this purpose.

09-15 17:55:52.039 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: 10%
09-15 17:55:52.100 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: 10%
09-15 17:55:52.102 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: onPageStarted : http://gmail.com/
09-15 17:55:52.102 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: onPageStarted
09-15 17:55:52.130 11065-12123/org.mozilla.rocket.debug.null E/mTwTm: Loading : http://gmail.com/
09-15 17:55:52.319 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: onPageStarted : https://www.google.com/gmail/
09-15 17:55:52.319 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: onPageStarted
09-15 17:55:52.935 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: onPageStarted : https://mail.google.com/mail/
09-15 17:55:52.935 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: onPageStarted
09-15 17:55:53.208 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: onPageStarted : https://accounts.google.com/ServiceLogin?service=mail&passive=true&continue=https://mail.google.com/mail/&ss=1&scc=1&ltmpl=ecobx&nui=5&btmpl=mobile&emr=1&osid=1#
09-15 17:55:53.208 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: onPageStarted
09-15 17:55:53.396 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: 20%
09-15 17:55:53.598 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: 25%
09-15 17:55:53.701 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: 70%
09-15 17:55:56.296 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: 90%
09-15 17:55:56.416 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: 100%
09-15 17:55:56.417 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: 100%
09-15 17:55:56.427 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: onPageFinished : https://accounts.google.com/ServiceLogin?service=mail&passive=true&continue=https://mail.google.com/mail/&ss=1&scc=1&ltmpl=ecobx&nui=5&btmpl=mobile&emr=1&osid=1#
09-15 17:55:56.446 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: onPageFinished
09-15 17:55:56.668 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: 10%
09-15 17:55:56.669 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: onPageFinished : https://accounts.google.com/ServiceLogin?service=mail&passive=true&continue=https://mail.google.com/mail/&ss=1&scc=1&ltmpl=ecobx&nui=5&btmpl=mobile&emr=1&osid=1#
09-15 17:55:56.685 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: onPageFinished
09-15 17:55:56.690 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: 100%
09-15 17:55:56.717 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: 10%
09-15 17:55:56.719 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: onPageFinished : https://accounts.google.com/ServiceLogin?service=mail&passive=true&continue=https://mail.google.com/mail/&ss=1&scc=1&ltmpl=ecobx&nui=5&btmpl=mobile&emr=1&osid=1#
09-15 17:55:56.724 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: onPageFinished
09-15 17:55:56.728 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: 100%
09-15 17:55:57.233 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: 10%
09-15 17:55:57.360 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: 55%
09-15 17:55:57.561 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: 66%
09-15 17:55:57.661 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: 70%
09-15 17:55:58.134 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: 85%
09-15 17:55:58.337 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: 95%
09-15 17:55:58.338 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: 100%
09-15 17:55:58.340 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: 100%
09-15 17:59:42.053 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: 10%
09-15 17:59:42.067 11065-12123/org.mozilla.rocket.debug.null E/mTwTm: Loading : https://accounts.google.com/TOS?loc=TW&hl=en&privacy=true
09-15 17:59:42.092 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: onPageStarted : https://accounts.google.com/TOS?loc=TW&hl=en&privacy=true
09-15 17:59:42.092 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: onPageStarted
09-15 17:59:42.185 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: onPageStarted : https://www.google.com.tw/intl/en/policies/privacy/
09-15 17:59:42.185 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: onPageStarted
09-15 17:59:42.510 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: 21%
09-15 17:59:42.709 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: 22%
09-15 17:59:42.833 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: 70%
09-15 17:59:43.817 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: 90%
09-15 17:59:44.568 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: 100%
09-15 17:59:44.568 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: 100%
09-15 17:59:44.569 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: onPageFinished : https://www.google.com.tw/intl/en/policies/privacy/
09-15 17:59:44.571 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: onPageFinished
09-15 18:00:26.022 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: 10%
09-15 18:00:26.027 11065-12123/org.mozilla.rocket.debug.null E/mTwTm: Loading : https://myaccount.google.com/?hl=en
09-15 18:00:26.060 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: onPageStarted : https://myaccount.google.com/?hl=en
09-15 18:00:26.061 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: onPageStarted
09-15 18:00:26.211 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: onPageStarted : https://myaccount.google.com/intro?hl=en
09-15 18:00:26.211 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: onPageStarted
09-15 18:00:27.025 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: 21%
09-15 18:00:27.154 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: 70%
09-15 18:00:29.623 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: onPageFinished : https://myaccount.google.com/intro?hl=en
09-15 18:00:29.631 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: onPageFinished
09-15 18:00:29.649 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: onPageFinished : https://myaccount.google.com/intro?hl=en
09-15 18:00:29.654 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: onPageFinished
09-15 18:00:31.768 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: 100%
09-15 18:00:31.789 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: 100%
09-15 18:00:31.789 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: onPageFinished : https://myaccount.google.com/intro?hl=en
09-15 18:00:31.797 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: onPageFinished
09-15 18:00:52.874 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: 10%
09-15 18:00:52.880 11065-12123/org.mozilla.rocket.debug.null E/mTwTm: Loading : https://myaccount.google.com/intro/preferences?hl=en#storage
09-15 18:00:52.892 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: onPageStarted : https://myaccount.google.com/intro/preferences?hl=en#storage
09-15 18:00:52.892 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: onPageStarted
09-15 18:00:53.143 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: 21%
09-15 18:00:53.222 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: 23%
09-15 18:00:53.307 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: 70%
09-15 18:00:56.386 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: 90%
09-15 18:00:56.387 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: onPageFinished : https://myaccount.google.com/intro/preferences?hl=en#storage
09-15 18:00:56.390 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: onPageFinished
09-15 18:00:56.412 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: onPageFinished : https://myaccount.google.com/intro/preferences?hl=en#storage
09-15 18:00:56.420 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: onPageFinished
09-15 18:00:57.409 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: 100%
09-15 18:00:57.410 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: 100%
09-15 18:00:57.411 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: onPageFinished : https://myaccount.google.com/intro/preferences?hl=en#storage
09-15 18:00:57.415 11065-11065/org.mozilla.rocket.debug.null E/mTwTm: onPageFinished

**Comment by mTwTm** [comment]
09-18 13:32:35.496 31933-31933/org.mozilla.rocket.debug.tyu E/mTwTm:   0 | android.widget.ProgressBar.<init>(ProgressBar.java:269)
09-18 13:32:35.581 31933-31933/org.mozilla.rocket.debug.tyu E/mTwTm:  10 | org.mozilla.focus.webkit.WebkitView$CallbackWrapper.onProgress(WebkitView.java:324)
09-18 13:32:35.682 31933-31933/org.mozilla.rocket.debug.tyu E/mTwTm:  10 | org.mozilla.focus.webkit.WebkitView$CallbackWrapper.onProgress(WebkitView.java:324)
09-18 13:32:35.688 31933-31933/org.mozilla.rocket.debug.tyu E/mTwTm:   5 | org.mozilla.focus.webkit.WebkitView$CallbackWrapper.onPageStarted(WebkitView.java:277)
09-18 13:32:35.907 31933-31933/org.mozilla.rocket.debug.tyu E/mTwTm:   5 | org.mozilla.focus.webkit.WebkitView$CallbackWrapper.onPageStarted(WebkitView.java:277)
09-18 13:32:35.995 31933-31933/org.mozilla.rocket.debug.tyu E/mTwTm:  20 | org.mozilla.focus.webkit.WebkitView$CallbackWrapper.onProgress(WebkitView.java:324)
09-18 13:32:36.118 31933-31933/org.mozilla.rocket.debug.tyu E/mTwTm:  70 | org.mozilla.focus.webkit.WebkitView$CallbackWrapper.onProgress(WebkitView.java:324)
09-18 13:32:36.170 31933-31933/org.mozilla.rocket.debug.tyu E/mTwTm: 100 | org.mozilla.focus.webkit.WebkitView$CallbackWrapper.onPageFinished(WebkitView.java:282)
09-18 13:32:36.303 31933-31933/org.mozilla.rocket.debug.tyu E/mTwTm: 100 | org.mozilla.focus.webkit.WebkitView$CallbackWrapper.onProgress(WebkitView.java:324)
09-18 13:32:36.306 31933-31933/org.mozilla.rocket.debug.tyu E/mTwTm: 100 | org.mozilla.focus.webkit.WebkitView$CallbackWrapper.onProgress(WebkitView.java:324)
09-18 13:32:36.309 31933-31933/org.mozilla.rocket.debug.tyu E/mTwTm: 100 | org.mozilla.focus.webkit.WebkitView$CallbackWrapper.onPageFinished(WebkitView.java:282)
09-18 13:32:54.960 31933-31933/org.mozilla.rocket.debug.tyu E/mTwTm:   0 | android.widget.ProgressBar.<init>(ProgressBar.java:269)
09-18 13:32:54.981 31933-31933/org.mozilla.rocket.debug.tyu E/mTwTm: 100 | android.view.View.dispatchRestoreInstanceState(View.java:15751)
09-18 13:32:55.004 31933-31933/org.mozilla.rocket.debug.tyu E/mTwTm:  10 | org.mozilla.focus.webkit.WebkitView$CallbackWrapper.onProgress(WebkitView.java:324)
09-18 13:32:55.251 31933-31933/org.mozilla.rocket.debug.tyu E/mTwTm:  70 | org.mozilla.focus.webkit.WebkitView$CallbackWrapper.onProgress(WebkitView.java:324)
09-18 13:32:55.252 31933-31933/org.mozilla.rocket.debug.tyu E/mTwTm: 100 | org.mozilla.focus.webkit.WebkitView$CallbackWrapper.onProgress(WebkitView.java:324)
09-18 13:32:55.252 31933-31933/org.mozilla.rocket.debug.tyu E/mTwTm: 100 | org.mozilla.focus.webkit.WebkitView$CallbackWrapper.onProgress(WebkitView.java:324)
09-18 13:32:55.253 31933-31933/org.mozilla.rocket.debug.tyu E/mTwTm:  10 | org.mozilla.focus.webkit.WebkitView$CallbackWrapper.onProgress(WebkitView.java:324)
09-18 13:32:55.254 31933-31933/org.mozilla.rocket.debug.tyu E/mTwTm: 100 | org.mozilla.focus.webkit.WebkitView$CallbackWrapper.onProgress(WebkitView.java:324)
09-18 13:32:55.254 31933-31933/org.mozilla.rocket.debug.tyu E/mTwTm: 100 | org.mozilla.focus.webkit.WebkitView$CallbackWrapper.onProgress(WebkitView.java:324)
09-18 13:32:55.255 31933-31933/org.mozilla.rocket.debug.tyu E/mTwTm:  10 | org.mozilla.focus.webkit.WebkitView$CallbackWrapper.onProgress(WebkitView.java:324)
09-18 13:32:55.262 31933-31933/org.mozilla.rocket.debug.tyu E/mTwTm:   5 | org.mozilla.focus.webkit.WebkitView$CallbackWrapper.onPageStarted(WebkitView.java:277)
09-18 13:32:56.255 31933-31933/org.mozilla.rocket.debug.tyu E/mTwTm:   5 | org.mozilla.focus.webkit.WebkitView$CallbackWrapper.onPageStarted(WebkitView.java:277)
09-18 13:32:56.846 31933-31933/org.mozilla.rocket.debug.tyu E/mTwTm:  20 | org.mozilla.focus.webkit.WebkitView$CallbackWrapper.onProgress(WebkitView.java:324)
09-18 13:32:56.944 31933-31933/org.mozilla.rocket.debug.tyu E/mTwTm:  23 | org.mozilla.focus.webkit.WebkitView$CallbackWrapper.onProgress(WebkitView.java:324)
09-18 13:32:58.052 31933-31933/org.mozilla.rocket.debug.tyu E/mTwTm:  23 | org.mozilla.focus.webkit.WebkitView$CallbackWrapper.onProgress(WebkitView.java:324)
09-18 13:32:58.195 31933-31933/org.mozilla.rocket.debug.tyu E/mTwTm:  30 | org.mozilla.focus.webkit.WebkitView$CallbackWrapper.onProgress(WebkitView.java:324)
09-18 13:32:58.338 31933-31933/org.mozilla.rocket.debug.tyu E/mTwTm:  60 | org.mozilla.focus.webkit.WebkitView$CallbackWrapper.onProgress(WebkitView.java:324)
09-18 13:32:58.440 31933-31933/org.mozilla.rocket.debug.tyu E/mTwTm:  78 | org.mozilla.focus.webkit.WebkitView$CallbackWrapper.onProgress(WebkitView.java:324)
09-18 13:32:58.543 31933-31933/org.mozilla.rocket.debug.tyu E/mTwTm:  82 | org.mozilla.focus.webkit.WebkitView$CallbackWrapper.onProgress(WebkitView.java:324)
09-18 13:32:58.644 31933-31933/org.mozilla.rocket.debug.tyu E/mTwTm:  85 | org.mozilla.focus.webkit.WebkitView$CallbackWrapper.onProgress(WebkitView.java:324)
09-18 13:32:58.779 31933-31933/org.mozilla.rocket.debug.tyu E/mTwTm:  85 | org.mozilla.focus.webkit.WebkitView$CallbackWrapper.onProgress(WebkitView.java:324)
09-18 13:32:59.556 31933-31933/org.mozilla.rocket.debug.tyu E/mTwTm:  85 | org.mozilla.focus.webkit.WebkitView$CallbackWrapper.onProgress(WebkitView.java:324)
09-18 13:32:59.655 31933-31933/org.mozilla.rocket.debug.tyu E/mTwTm:  87 | org.mozilla.focus.webkit.WebkitView$CallbackWrapper.onProgress(WebkitView.java:324)
09-18 13:32:59.825 31933-31933/org.mozilla.rocket.debug.tyu E/mTwTm:  92 | org.mozilla.focus.webkit.WebkitView$CallbackWrapper.onProgress(WebkitView.java:324)
09-18 13:33:00.118 31933-31933/org.mozilla.rocket.debug.tyu E/mTwTm:  93 | org.mozilla.focus.webkit.WebkitView$CallbackWrapper.onProgress(WebkitView.java:324)
09-18 13:33:00.246 31933-31933/org.mozilla.rocket.debug.tyu E/mTwTm:  94 | org.mozilla.focus.webkit.WebkitView$CallbackWrapper.onProgress(WebkitView.java:324)
09-18 13:33:00.704 31933-31933/org.mozilla.rocket.debug.tyu E/mTwTm:  95 | org.mozilla.focus.webkit.WebkitView$CallbackWrapper.onProgress(WebkitView.java:324)
09-18 13:33:01.123 31933-31933/org.mozilla.rocket.debug.tyu E/mTwTm:  96 | org.mozilla.focus.webkit.WebkitView$CallbackWrapper.onProgress(WebkitView.java:324)
09-18 13:33:01.286 31933-31933/org.mozilla.rocket.debug.tyu E/mTwTm:  98 | org.mozilla.focus.webkit.WebkitView$CallbackWrapper.onProgress(WebkitView.java:324)
09-18 13:33:01.330 31933-31933/org.mozilla.rocket.debug.tyu E/mTwTm: 100 | org.mozilla.focus.webkit.WebkitView$CallbackWrapper.onProgress(WebkitView.java:324)
09-18 13:33:01.331 31933-31933/org.mozilla.rocket.debug.tyu E/mTwTm: 100 | org.mozilla.focus.webkit.WebkitView$CallbackWrapper.onProgress(WebkitView.java:324)
09-18 13:33:01.333 31933-31933/org.mozilla.rocket.debug.tyu E/mTwTm: 100 | org.mozilla.focus.webkit.WebkitView$CallbackWrapper.onPageFinished(WebkitView.java:282)
09-18 13:33:35.613 31933-31933/org.mozilla.rocket.debug.tyu E/mTwTm:   0 | android.widget.ProgressBar.<init>(ProgressBar.java:269)
09-18 13:33:35.631 31933-31933/org.mozilla.rocket.debug.tyu E/mTwTm: 100 | android.view.View.dispatchRestoreInstanceState(View.java:15751)
09-18 13:33:35.654 31933-31933/org.mozilla.rocket.debug.tyu E/mTwTm:  10 | org.mozilla.focus.webkit.WebkitView$CallbackWrapper.onProgress(WebkitView.java:324)
09-18 13:33:35.898 31933-31933/org.mozilla.rocket.debug.tyu E/mTwTm:  10 | org.mozilla.focus.webkit.WebkitView$CallbackWrapper.onProgress(WebkitView.java:324)
09-18 13:33:35.899 31933-31933/org.mozilla.rocket.debug.tyu E/mTwTm: 100 | org.mozilla.focus.webkit.WebkitView$CallbackWrapper.onProgress(WebkitView.java:324)
09-18 13:33:35.900 31933-31933/org.mozilla.rocket.debug.tyu E/mTwTm: 100 | org.mozilla.focus.webkit.WebkitView$CallbackWrapper.onProgress(WebkitView.java:324)
09-18 13:33:35.901 31933-31933/org.mozilla.rocket.debug.tyu E/mTwTm:  10 | org.mozilla.focus.webkit.WebkitView$CallbackWrapper.onProgress(WebkitView.java:324)
09-18 13:33:35.911 31933-31933/org.mozilla.rocket.debug.tyu E/mTwTm:   5 | org.mozilla.focus.webkit.WebkitView$CallbackWrapper.onPageStarted(WebkitView.java:277)
09-18 13:33:36.245 31933-31933/org.mozilla.rocket.debug.tyu E/mTwTm:  20 | org.mozilla.focus.webkit.WebkitView$CallbackWrapper.onProgress(WebkitView.java:324)
09-18 13:33:36.357 31933-31933/org.mozilla.rocket.debug.tyu E/mTwTm:  49 | org.mozilla.focus.webkit.WebkitView$CallbackWrapper.onProgress(WebkitView.java:324)
09-18 13:33:36.821 31933-31933/org.mozilla.rocket.debug.tyu E/mTwTm:  69 | org.mozilla.focus.webkit.WebkitView$CallbackWrapper.onProgress(WebkitView.java:324)
09-18 13:33:36.921 31933-31933/org.mozilla.rocket.debug.tyu E/mTwTm:  90 | org.mozilla.focus.webkit.WebkitView$CallbackWrapper.onProgress(WebkitView.java:324)
09-18 13:33:37.377 31933-31933/org.mozilla.rocket.debug.tyu E/mTwTm:  93 | org.mozilla.focus.webkit.WebkitView$CallbackWrapper.onProgress(WebkitView.java:324)
09-18 13:33:37.530 31933-31933/org.mozilla.rocket.debug.tyu E/mTwTm:  95 | org.mozilla.focus.webkit.WebkitView$CallbackWrapper.onProgress(WebkitView.java:324)
09-18 13:33:37.691 31933-31933/org.mozilla.rocket.debug.tyu E/mTwTm:  95 | org.mozilla.focus.webkit.WebkitView$CallbackWrapper.onProgress(WebkitView.java:324)
09-18 13:33:37.830 31933-31933/org.mozilla.rocket.debug.tyu E/mTwTm:  98 | org.mozilla.focus.webkit.WebkitView$CallbackWrapper.onProgress(WebkitView.java:324)
09-18 13:33:37.861 31933-31933/org.mozilla.rocket.debug.tyu E/mTwTm: 100 | org.mozilla.focus.webkit.WebkitView$CallbackWrapper.onProgress(WebkitView.java:324)
09-18 13:33:37.868 31933-31933/org.mozilla.rocket.debug.tyu E/mTwTm: 100 | org.mozilla.focus.webkit.WebkitView$CallbackWrapper.onProgress(WebkitView.java:324)
09-18 13:33:37.876 31933-31933/org.mozilla.rocket.debug.tyu E/mTwTm: 100 | org.mozilla.focus.webkit.WebkitView$CallbackWrapper.onPageFinished(WebkitView.java:282)

Pasting a simpler log:
In short, there are two things in play that interfere our progress bar loading:
1) We should not assume onPageStarted and onPageFinished as 0 and 100 respectively. 
2) There are some incomplete onPageFinished fired everytime we try to resume webview state (Also see: 1d055d3b67a25feb2777473d908c7a915c8f1d2d) We will see progress updates of the resumed page. This should be filtered out.

**Comment by cssjh** [comment]
**Verified** on 1267.
